syncer/syncer.go

```diff
@@ -1574,6 +1574,8 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 				if err != nil {
 					return errors.Trace(err)
 				}
+				target, _ := GenTableID(ddlInfo.tableNames[1][0].Schema, ddlInfo.tableNames[1][0].Name)
+				unsyncedTableGauge.WithLabelValues(s.cfg.Name, target).Set(float64(remain))
```

Comment:
why not put it into L1582 scope

Reply:
this scope can be reached only in sharding mode, see `continue` in L1542

Reply:
but I'll move it to L1582.

Reply:
in order to handle `CreateTableStmt` in L1565, so we move it into L1582 scope?

Reply:
addressed

Reply:
yep, and we also have non-shard table in shard mode